.gitignore

```diff
@@ -6,5 +6,4 @@
 *.ipynb_checkpoints
 Manifest.toml
 .vscode
-*.h5
```

Comment:
@scdhulipala why did you remove this from the .gitignore. It is not a good practice to upload binary files to Github

Reply:
I reverted this change.